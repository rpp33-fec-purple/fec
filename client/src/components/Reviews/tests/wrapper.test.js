// import React from 'react';
// import TestRenderer from 'react-test-renderer';
// import Review from '../wrapper';

// describe('render the component Reviews', () => {
// const testRenderer = TestRenderer.create(<Review />);
// const testInst = testRenderer.root;
// it('should contain a div element ', () => {
//   expect(testRenderer.toJSON().type).toBe('div');
// })

// it('should return the right inner text of the existing element ', () => {
//   expect(testInst.findByProps({className:'ratingsAndReviews'}).children[0]).toEqual('Review Component');
// })

// it('the component should match the saved snapshot', () => {
//   expect(testRenderer.toJSON()).toMatchSnapshot();
// })

// })
import React from 'react';
import { render, screen } from '@testing-library/react';
import Wrapper from '../wrapper.jsx';

test('should render a div with className ratingsAndReviews', () => {
  render(<Wrapper/>)

  screen.debug();
})

