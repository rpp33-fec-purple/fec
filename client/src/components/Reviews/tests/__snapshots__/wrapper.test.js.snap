// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the component Reviews the component should match the saved snapshot 1`] = `
<div
  className="ratingsAndReviews"
>
  <h4
    className="sc-hGPBjI huxpbn"
  >
    RATINGS & REVIEWS
  </h4>
  <div
    className="sc-dJjYzT bnMCNC"
  >
    <div />
    <div>
      <button
        className="sc-egiyK boQJXe"
        onClick={[Function]}
      >
        ADD A REVIEW +
      </button>
    </div>
  </div>
</div>
`;
